packaging/nfpm/func-e.8

```diff
@@ -1,47 +1,47 @@
 .nh
-.TH func\-e 8
```

Comment:
some indirect dep escaped incorrectly! only control characters should have been escaped